src/solvers/dg/dg_2d.jl

```diff
@@ -828,23 +828,31 @@ function prolong2mortars!(cache, u::AbstractArray{<:Any,4}, equations,
       leftright = 1
       if cache.mortars.orientations[mortar] == 1
         # L2 mortars in x-direction
-        u_large = view(u, :, nnodes(dg), :, large_element)
-        element_solutions_to_mortars!(cache, mortar_l2, leftright, mortar, u_large)
+        # u_large = view(u, :, nnodes(dg), :, large_element)
+        # element_solutions_to_mortars!(cache, mortar_l2, leftright, mortar, u_large)
+        element_solutions_to_mortars!(cache, mortar_l2, leftright, mortar,
+                                      view(u, :, nnodes(dg), :, large_element))
```

Comment:
Is this really just moving the view inside the function argument list? What is the advantage? And why keep the old code commented? Sorry, I am confused...

Reply:
`@code_warntype` complained about the previous version. I will revisit this part and remove outdated code. This seems to be a leftover from me programming when I was tired...

Reply:
I reverted these changes in 4fbf359a789fabdf058ff6bfa1c479e9390b171c since more benchmarking of `Trixi.rhs!` didn't show significant differences.